README.md

```diff
@@ -62,15 +90,6 @@ npm run copy:remote-production
 ## iOS
 
 ```bash
-# install cocoapods
-sudo gem install cocoapods
-# install local dependencies
-npm install
-
-cd ios
-pod install
-cd ..
-
 # Run a simulator pointed at a static build
 npm run ios
 # Run a simulator pointed at localhost
```

Comment:
```suggestion
# Run a simulator pointed at URL_OVERRIDE
```

Reply:
I think we should revisit the comments in this section because `npm run ios` vs `npm run ios:dev` isn't necessarily static build vs localhost. It's just using different .env files